lib/connection/connection_config.js

```diff
@@ -948,7 +948,7 @@ function createParameters()
     },
     {
       name: PARAM_RETRY_SF_MAX_LOGIN_RETRIES,
-      defaultValue: 5,
+      defaultValue: 7,
```

Comment:
This looks correct, I'm just wondering about if max wait time is already 300 and that's why there's no change

Reply:
I could not find the max time out in the parameters. There is a parameter called sfRetryMaxSleepTime, but it is the max time for sleep, not the total time. I am not sure whether I need to make it or not.

Reply:
If the max wait time is supposed to be customizable for the user then you may have to make one for it